tests/unit_tests/gui/test_csv_export.py

```diff
@@ -55,6 +56,10 @@ def handle_finished_box():
     QTimer.singleShot(3000, handle_finished_box)
     export_csv_action.trigger()
 
+    runner = plugin_tool.get_plugin_runner("CSV Export")
```

Comment:
Shall we just `assert runner.poll_thread is not None`? Or it can be `None`?

Reply:
Can probably not be None. Ill change to assert